.golangci.yaml

```diff
@@ -113,6 +113,12 @@ issues:
     - '(Expect directory permissions to be 0750 or less|Expect (WriteFile|file) permissions to be 0600 or less)'
 
   exclude-rules:
+    - path: pinniped/internal/mocks
+      linters:
+        - goimports
+    - path: apis/run/v1alpha1
+      linters:
+        - goimports
```

Comment:
These are not valid paths anymore.

Reply:
Good catch. Sorry about the rebase. I shoud've put the change in a separate commit, but got carried away with clean up. These paths are no longer in the rebased PR.